examples/solids/README.rst

```diff
@@ -198,3 +198,43 @@ For example, consider a problem involving metals subject to gravity.
 
 One can choose units of displacement independently (e.g., :code:`-units_meter 100` to measure displacement in centimeters), but :math:`E` and :math:`\int \rho \bm g` have the same dependence on mass and time, so cannot both be made of order 1.
 This reflects the fact that both quantities are not equally significant for a given displacement size; the relative significance of gravity increases as the domain size grows.
+
+Diagnostic Quantities
+^^^^^^^^^^^^^^^^^^^^^
+
+Diagnostic quantities for viewing are provided when the command line options for visualization output, :code:`-view_soln` or :code:`-view_final_soln` are used.
+The diagnostic quantities include displacement in the :math:`x` direction, displacement in the :math:`y` direction, displacement in the :math:`z` direction, pressure, :math:`\operatorname{trace} \left( E \right)`, :math:`\operatorname{trace} \left( E^2 \right)`, :math:`\lvert J \rvert`, and strain energy density.
+The table below summarizes the formulations of each of these quantities for each problem type.
+
+.. list-table:: Diagnostic quantities
+   :header-rows: 1
+
+   * - Quantity
+     - Linear Elasticity
+     - Hyperelasticity, Small Strain
+     - Hyperelasticity, Finite Strain
+
+   * - Pressure
+     - :math:`\lambda \sum_i \bm{\epsilon}_{ii}`
+     - :math:`\lambda \log \left( \sum_i \bm{\epsilon}_{ii} \right)`
+     - :math:`\lambda \log J`
+
+   * - :math:`\operatorname{trace} \left( E \right)`
+     - :math:`\sum_i \bm{\epsilon}_{ii}`
+     - :math:`\sum_i \bm{\epsilon}_{ii}`
+     - :math:`\sum_i E_{ii}`
```

Comment:
Why the different notation?

Reply:
Between line 222 and the rest, or line 223/224 and 225?

Reply:
I think he meant that as you have bold face for small `epsilon`, you should have it for `E` as well, since they are both tensors

Reply:
There is another `\sum_i E_{ii}` on line 241

Reply:
The customary notation is actually to use bold face when referring to the tensor itself, but non bold face, when you indicate the trace in index notation. Since in that case, you are referring to the sum of the single entries

Reply:
See [for instance](https://en.wikipedia.org/wiki/Trace_(linear_algebra))

Reply:
Why not use `\operatorname{trace} \bm E` style notation in all cases?

Reply:
To me it looks strange to have the heading of the row be the same as the contents. I have update the table to exclusively use `\operatorname{trace} \bm E` or `\operatorname{trace} \bm{\epsilon}`.

Reply:
You can change the row heading for instance to volumetric strain, or volume change

Reply:
Please also check that the documentation compiles both with `html` and `latexpdf` targets. It now breaks with the following error:
```
Runaway argument?
! Forbidden control sequence found while scanning use of \bm.
<inserted text> 
                \par 
l.2462 ...e, \(\operatorname{trace} \left( \bm \bm
                                                  {E} \right)\), \(\operator...

```

Reply:
It wasn't clear to me that in general the interpretation of the Green-Lagrange strain tensor invariants exactly matched the interpretation of the linear strain tensor, even though the quantities have similar computation and the Green-Lagrange strain tensor converges to the linear strain tensor in the small deformation limit. That's why I stuck with the more general description of the invariants.

Reply:
See the [Physical Interpretation of Invariants note](https://www.continuummechanics.org/principalstrain.html). 
PS: I found this website really useful when I was doing Continuum Mechanics in my PhD